frontend/server/src/DAO/Problems.php

```diff
@@ -829,37 +831,39 @@ public static function getAdminUser(\OmegaUp\DAO\VO\Problems $problem): ?array {
     }
 
     /**
-     * @return list<\OmegaUp\DAO\VO\Problems>
+     * @return array{problems: list<\OmegaUp\DAO\VO\Problems>, total: int}
      */
-    public static function getAllWithTotal(
+    public static function getAllWithCount(
         int $page,
-        int $pageSize,
-        int &$total
+        int $pageSize
     ) {
-        /** @var int $total */
+        /** @var int */
         $total = \OmegaUp\MySQLConnection::getInstance()->GetOne(
-            'SELECT COUNT(*) FROM `Problems`',
-            []
+            'SELECT COUNT(*) FROM `Problems`'
         );
 
-        return \OmegaUp\DAO\Problems::getAll(
+        $problems = \OmegaUp\DAO\Problems::getAll(
             $page,
             $pageSize,
             'problem_id',
             'DESC'
         );
+
+        return [
+            'problems' => $problems,
+            'total' => $total,
```

Comment:
nit:

```suggestion
            'count' => $count,
```

igual en todos los demás lugares donde se usa "total"

Reply:
Listo